odoo/addons/base/models/res_users.py

```diff
@@ -36,11 +36,6 @@ class CryptContext:
     def __init__(self, *args, **kwargs):
         self.__obj__ = _CryptContext(*args, **kwargs)
 
-    @property
-    def encrypt(self):
-        # deprecated alias
-        return self.hash
```

Comment:
Apparently created with https://github.com/odoo/odoo/commit/9dbb26a0b20634998a0b8a44fc86e3898feb0431 -> do you know why that deprecated alias was created? There might be a reason not to remove it?

@mwath ?